### DISCUSSION TITLE
(SUGGESTION)

### DISCUSSION THREAD

**Discussion by johnsoncui02** [Zrips/CMI#1847] [Enhancement]
- **add a chance system for attached commands**. so there's a percentage chance to execute a random command within all of the list of commands perhaps.

- add a playerwarp function where it will allow players to create their own warps. (separate GUI from admin set warps)

**Comment by Drc-DEV** [comment]
Players can already set homes, why have warps?

**Comment by mrfloris** [comment]
> Players can already set homes, why have warps?

Loads of our build servers for example use /visit, as a way to set a start point of a base, or adventure, or whatever they made. It's a way for everybody to know: Oh, maybe i can just type /visit player, and see if i can explore there.. 

Players aren't always online, and warps are usually limited to server stuff or exclusive to higher paying ranks. 

That said, you can use CMI's custom alias system to create /visit and give perms to players for their own warp creation. Using teh alias you can also force it to be on say page 4.. so it doesn't go to page 1. But of course, this has some limitations.

**Comment by johnsoncui02** [comment]
Caveat on what floris said. What I meant by player warps is to create an additional warping system that will allow players to create Public Warps which will let anyone on the server to use for. This is very helpful for players who enjoy showing off and "flexing" on other players with the builds they've made. It also helps setup and simplify things such as creation shops where players create their own shop. While it is a player shop it does not conflict with the "admin warps" that are set default for the purpose of guiding new comers. Thank you :)
